bitbots_blackboard/bitbots_blackboard/capsules/costmap_capsule.py

```diff
@@ -69,13 +69,15 @@ def robot_callback(self, msg: RobotArray):
         """
         Callback with new obstacles
         """
+        self._blackboard.node.get_logger().warn("entering method")
```

Comment:
Did you push the entering and leaving warnings intentionally?

Reply:
No. I thought i removed them all in [9a5c13f](https://github.com/bit-bots/bitbots_behavior/pull/281/commits/9a5c13fbd6debc70454ab123182d2cf11c8f05ea), but apparently I did not

Reply:
You are correct. Apparently, I looked at the changes from an intermediate commit. Don't know why. Sorry for causing the confusion.